blog/apps.py

```diff
@@ -0,0 +1,10 @@
+"""This module configures the Blog application."""
```

Comment:
Same as above. A module is not an entry point. It contains declarations but it shouldn't generally have side effects.

Reply:
Edited.

Reply:
Marking as resolved.